tests/Unit/Infrastructure/Auth/SentinelAccountManagementTest.php

```diff
@@ -85,7 +85,7 @@ public function testFindByRoleReturnsArray()
         $sentinel = Mockery::mock(Sentinel::class);
         $sentinel->shouldReceive('getRoleRepository->findByName->getUsers->toArray')->andReturn([]);
         $accounts = new SentinelAccountManagement($sentinel);
-        $this->assertEquals([], $accounts->findByRole('blabla'));
+        $this->assertSame([], $accounts->findByRole('blabla'));
```

Comment:
Since [`AccountManagement::findByRole()`](https://github.com/opencfp/opencfp/blob/83c71ba09618c0fe5bab46facb5737cb57b073c7/classes/Domain/Services/AccountManagement.php#L24) has `array` as return type declaration, we can probably just as well use `assertEmpty()` here, what do you think?

Reply:
Sound good to me, but there are probably more cases like that, so i prefer to do all those in one (later) PR